### PR TITLE
Task 3. Sum of Range

### PR DESCRIPTION
**Pull request by LeonidShatilo**
- Implement function `range(min, max)` that can return sum of integer numbers in range [min, max].
- Consider edge cases: result is greater than `Number.MAX_SAFE_INTEGER`, argument is not a number, min > max, etc.
- **\* An additional assignment:** For performance reason add memoization mechanism.

### CODE REVIEW

File: task-03-sum-of-range/src/scripts/listener.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+INPUTS.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
+    if (isMinError || isMaxError) {
+      return;
+    }
+    summing();
+  }
+});
```
Comment (broshkabro) [inline task-03-sum-of-range/src/scripts/listener.js:33]: I think it is better to combine these conditions, however, you need to change the 2nd condition for it.

**Review by broshkabro — APPROVED** [review]
awesome!

File: task-03-sum-of-range/src/scripts/errorHandling.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+export function removeErrorSettings(input) {
+  if (input === 'min') {
+    isMinError = false;
+  }
+  if (input === 'max') {
+    isMaxError = false;
+  }
+  if (!isMinError && !isMinError) {
```
[the remark this answers is not in the document]
Comment (LeonidShatilo) [reply task-03-sum-of-range/src/scripts/errorHandling.js]: Thanks, I didn't notice that inaccuracy! Сorrected it.

File: task-03-sum-of-range/src/scripts/listener.js (hunk 1 above)
Comment (LeonidShatilo) [reply task-03-sum-of-range/src/scripts/listener.js:32]: Totally agree. I combined the conditions into one.

**Pull request merged**